scripts/prebuild

```diff
@@ -155,17 +112,24 @@ var downloads =
   }
 ]
 
-if(process.platform === 'darwin')
-{
-  downloads[2].patch = LINUX_PATCH_PATH
+// Copy musl headers for compilation on OS X
+function copy_headers(callback) {
```

Comment:
Please put the function in the same place it was before, so the diff is smaller and easier to compare.